.env.example

```
@@ -23,6 +23,7 @@ VATSCA_MEMBER_ROLE = 0
 VATSIM_MEMBER_ROLE = 0
 EVENTS_ROLE = 0
 MENTOR_ROLE = 0
+BUDDY_ROLE = 0
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)